src/codecovio.jl

```diff
@@ -143,15 +143,15 @@ module Codecov
     import Base.LibGit2
 
     """
-        submit_local(fcs::Vector{FileCoverage})
+        submit_local(fcs::Vector{FileCoverage}, dir::AbstractString=pwd())
 
-    Takes a `Vector` of file coverage results (produced by `process_folder`),
-    and submits them to Codecov.io. Assumes the submission is being made from
-    a local git installation.  A repository token should be specified by a
-    'token' keyword argument or the `CODECOV_TOKEN` environment variable.
+    Take a `Vector` of file coverage results (produced by `process_folder`),
+    and submit them to Codecov.io. Assumes the submission is being made from
+    a local git installation, rooted at `dir`. A repository token should be specified by a
+    `token` keyword argument or the `CODECOV_TOKEN` environment variable.
     """
-    function submit_local(fcs::Vector{FileCoverage}; kwargs...)
-        LibGit2.with(LibGit2.GitRepo(pwd())) do repo
+    function submit_local(fcs::Vector{FileCoverage}, dir::AbstractString=pwd(); kwargs...)
```

Comment:
Should `joinpath(pwd(), "..")` be the default if that's what's needed to get the tests to work?

Reply:
Does `Pkg.test` run things in `PKG_ROOT/test` or `PKG_ROOT`?

Reply:
I could be mistaken, but looking at the code it appears to run in `PKG_ROOT/test`.